livre/eq/render/RayCastRenderer.cpp

```diff
@@ -538,8 +550,44 @@ struct RayCastRenderer::Impl
         glUseProgram( 0 );
     }
 
-    void onFrameEnd()
+    void renderAxis( const Frustum& frustum )
+    {
+        GLSLShaders::Handle program = _axisShaders.getProgram( );
+        LBASSERT( program );
+        glUseProgram( program );
+
+        GLint tParamNameGL = glGetUniformLocation( program, "renderTexture" );
```

Comment:
It's everywhere like that... Unless I change it everywhere in Livre I will keep it like that